src/text-editor.coffee

```diff
@@ -609,15 +609,19 @@ class TextEditor extends Model
   # * "<filename>" when it is the only editing buffer with this file name.
   # * "<filename> — <unique-dir-prefix>" when other buffers have this file name.
   #
+  # * `textEditors` An optional {Array} of the other {TextEditor} instances that
```

Comment:
I agree, separating TextEditor/TextEditorPaneItem makes sense 👍 

The surface area for deprecation makes me nervous. It's gonna be annoying for ... a lot of packages. Do we have much president for this sort of large change?